src/spatialite.mk

```diff
@@ -8,11 +8,11 @@ $(PKG)_VERSION  := 4.4.0-RC1
 $(PKG)_CHECKSUM := 80f7fff0a147044c5eb197e565f598ac1f137d86d0a548cbc8f52fb7ff7cac68
 $(PKG)_SUBDIR   := libspatialite-$($(PKG)_VERSION)
 $(PKG)_FILE     := libspatialite-$($(PKG)_VERSION).tar.gz
-$(PKG)_URL      := http://www.gaia-gis.it/gaia-sins/libspatialite-sources/$($(PKG)_FILE)
+$(PKG)_URL      := https://www.gaia-gis.it/gaia-sins/libspatialite-sources/$($(PKG)_FILE)
 $(PKG)_DEPS     := gcc dlfcn-win32 freexl geos libiconv libxml2 proj sqlite zlib
 
 define $(PKG)_UPDATE
-    $(WGET) -q -O- 'http://www.gaia-gis.it/gaia-sins/libspatialite-sources/' | \
+    $(WGET) -q -O- 'https://www.gaia-gis.it/gaia-sins/libspatialite-sources/' | \
```

Comment:
```
Forbidden

You don't have permission to access /gaia-sins/libspatialite-sources/ on this server.

Apache/2.2.15 (CentOS) Server at www.gaia-gis.it Port 443
```

Reply:
My mistake. It's a weird server config though!